charts/opentelemetry-collector/examples/daemonset-collector-logs/values.yaml

```diff
@@ -7,3 +7,9 @@ config:
   receivers:
     filelog:
       exclude: []
+  service:
+    pipelines:
+      logs:
+        receivers:
+          - otlp
+          - filelog
```

Comment:
Once presets are done properly in https://github.com/open-telemetry/opentelemetry-helm-charts/issues/158, this whole config should not be needed anymore